flash/core/trainer.py

```diff
@@ -79,7 +79,6 @@ def __init__(self, *args, **kwargs):
         if flash._IS_TESTING:
             if torch.cuda.is_available():
                 kwargs["gpus"] = -1
-                kwargs["max_epochs"] = 3
```

Comment:
Is there a reason for this removal?

Reply:
Yeah, it was forcing 3 epochs in the test but video classification takes too long so was timing out. Now each example sets it's own max epochs and we just use that 😃

Reply:
Awesome! Thanks for the explanation.